components/Footer/index.js

```diff
@@ -69,9 +68,28 @@ const StyledNav = styled.nav`
   position: fixed;
   bottom: 0;
   align-items: center;
-  padding: 0;
+
   width: 100%;
-  background-color: #097969;
-  border-radius: 10px;
-  height: 70px;
+  background-color: var(--secondary-color);
+
+  height: 75px;
+`;
+
+const StyledParagraph = styled.p`
+  font-size: 14px;
+  font-weight: bold;
+  letter-spacing: 0;
+  text-align: center;
+  margin: 0;
+`;
+
+const StyledImage = styled(Image)`
+  &.active {
+    opacity: 1;
+  }
+  &.notActive {
+    opacity: 0.3;
+  }
 `;
```

Comment:
This the perfect case to style the component dynamically with props. 🤩

```suggestion
  const StyledImage = styled(Image)`
  opacity: ${({ variant }) => (variant === 'active' ? 1 : 0.3)};
`;
```